### ISSUE TITLE
Mark all selected diffs as reviewd

### ISSUE THREAD

**Issue by sfc-gh-lmoll** [MarcinZukowski/review-pal#1]
Sometimes a PR contains lots of small subsequent diffs, e.g. when some object was renamed. It would be handy if I could simply select all of them (already possible) and then mark all selected diffs as reviewed with a single click / shortcut.

**Comment by MarcinZukowski** [comment]
Not sure if  I understand correctly. Do you want to select using a browser selection mechanism, and then somehow mark the overlapping (or fully contained?) diffs as marked?

For now, you can mark an entire file as reviewed, that should also help.

**Comment by sfc-gh-lmoll** [comment]
Yes, imagine a file with the following changed lines (only mentioning changed lines here, imagine there is lots of unchanged code in between all lines):
```
fooRenamed
fooRenamed
fooRenamed
fooRenamed
some-significant-change
fooRenamed
fooRenamed
fooRenamed
fooRenamed
some-other-significant-change
```

When I go through the file I will manually clear all lines that only contain the renaming of `foo` as diff. After that I will have an overview of the really important changes:

```
some-significant-change
some-other-significant-change
```

It would help if I wouldn't have to click a button or press `shift+K shift+X` once for every `fooRenamed`.
There already is a way to select multiple lines in Github:


E.g. if lines are selected like that the _Mark all done_ and _Clear all_ buttons as well as `shift+X` could be applied only to the selected lines.


Marking an entire helps a bit, I could mark it all as done and then manually search for significant changes and unmark them again. But that's a blacklisting vs. whitelisting approach.
Also it doesn't work for the first file for me, because the Review-Pal toolbar hides the button:

But I think this is only because of my very small display resolution atm, hope a normal sized monitor will fix this.

**Comment by MarcinZukowski** [comment]
I think it would be possible to detect which diff lines are a part of selection, and indeed mark them as reviewed. This could require splitting the diff into a sub-diff, but can be done. 

When I find more time, I'll take a look
